descriptors/ModuleDescriptor-template.json

```diff
@@ -446,7 +446,7 @@
     },
     {
       "permissionName": "change-manager.parsedrecords.get",
-      "displayName": "Change Manager - get parsed records by instanceId",
+      "displayName": "Change Manager - get parsed records by externalId",
```

Comment:
Update major interface version of interface with this endpoint

Reply:
@natali-zaitseva, what we should do in case of a major version change?

Reply:
Thank you!
As I can see only mod-quick-marc and ui-agreements are using this interface. 
![image](https://user-images.githubusercontent.com/5904625/131660235-3923681b-ec16-433c-b344-d1a7d02b2c8d.png)

Reply:
and we should it before R3 "Major interface provided to be updated", which seems to be before 17 Sep.

Reply:
I notified ERM team directly about the change. But we still could notify #development channel.